appendix.ru.md

```diff
@@ -2,23 +2,23 @@
 lang: ru
 ---
 
-Ричард М. Столлман, известный своим негативным отношением к женщинами, заставлял их чувствовать себя некомфортно и уязвимо. Информацию об инцидентах, связанных с Ричардом М. Столлманом и MIT, см. в [0].
+Ричард М. Столлман, известный своим неподобающим поведением в отношении к женщинам, заставлял их чувствовать себя некомфортно и уязвимо. Информацию об инцидентах, связанных с Ричардом М. Столлманом и MIT, см. в [0].
```

Comment:
*mistreating women* is probably better translated as *inappropriate behaviours towards women* rather than *negative attitude to women* which is what the current text translates to.

Reply:
"неподобающее поведение" это довольно обширное понятие, под которое можно подвести что угодно. В данном случае идёт речь конкретно о поведении с знаком "минус".

Reply:
The original text doesn’t say anything about *negative behaviour*. It says *mistreating* which is not *negative behaviour*/*negative attitude*. *Mistreating* is *неподобающее поведение*.

Reply:
Ну уж давайте не будем раздувать из мухи слона, для слова "неподобающее поведение" есть перевод "inappropriate behavior", то что вы показали - абсолютная неточность перевода.

Reply:
Translation is not 1:1 mapping. Sometimes you need to use a different phrase to preserve the meaning.

Reply:
"негативным отношением" ("negative") is definitely inaccurate and distorts the meaning from mistreating (i.e. wrongful actions) to misogyny (i.e. wrongful beliefs).  If you want a stronger and more accurate synonym, I propose "недопустимым поведением" ("unacceptable behavior").